--- qof-0.6.1/configure.in.orig	2005-12-29 17:35:14.000000000 +0100
+++ qof-0.6.1/configure.in	2006-02-21 23:44:48.046481500 +0100
@@ -119,8 +119,8 @@
 
 GDA_REQUIRED=1.2.0
 AC_MSG_CHECKING(for GDA)
-LIBGDA_CFLAGS=`$PKG_CONFIG --silence-errors --cflags "libgda >= $GDA_REQUIRED"`
-LIBGDA_LIBS=`$PKG_CONFIG --silence-errors --libs "libgda >= $GDA_REQUIRED"`
+LIBGDA_CFLAGS=`$PKG_CONFIG --silence-errors --cflags "libgda-2.0 >= $GDA_REQUIRED"`
+LIBGDA_LIBS=`$PKG_CONFIG --silence-errors --libs "libgda-2.0 >= $GDA_REQUIRED"`
 AC_SUBST(LIBGDA_CFLAGS)
 AC_SUBST(LIBGDA_LIBS)
 if test "x$LIBGDA_LIBS" = x; then
@@ -129,7 +129,7 @@
 	ADD_PKG_LIB="-lqofsql"
 	AC_SUBST(LIBQOFSQL_LIBRARY_VERSION)
 else
-    GDA_VERSION=`$PKG_CONFIG --modversion libgda`
+    GDA_VERSION=`$PKG_CONFIG --modversion libgda-2.0`
     AC_MSG_RESULT([yes, using GDA: ${GDA_VERSION}])
     AC_DEFINE(HAVE_GDA,,[We have found GDA])
 	ADD_PKG_LIB="-lgda-2"
