--- qof-0.7.5/configure.in.orig	2008-05-30 23:35:01.663046372 +0200
+++ qof-0.7.5/configure.in	2008-05-30 23:34:43.125992519 +0200
@@ -264,7 +264,7 @@
 fi
 if test "x$LIBGDA_CFLAGS" = x; then
     AC_MSG_RESULT([GDA not found (or disabled); will build using internal SQL library.])
-    SQL_PKG_LIB="-lqofsql"
+    SQL_PKG_LIB="\$(top_builddir)/lib/libsql/libqofsql.la"
 else
     AC_ARG_ENABLE(gdabackend,
       [  --enable-gdabackend     Enable the use of the gda backend (off).],
--- qof-0.7.5/qof/Makefile.am.orig	2008-01-14 20:50:54.000000000 +0100
+++ qof-0.7.5/qof/Makefile.am	2008-05-30 23:33:57.747405703 +0200
@@ -3,7 +3,7 @@
 lib_LTLIBRARIES = libqof.la
 
 libqof_la_LDFLAGS= -version-info $(LIBQOF_LIBRARY_VERSION) \
-  $(LINKER_SCRIPT) -L${top_srcdir}/lib/libsql
+  $(LINKER_SCRIPT)
 
 libqof_la_SOURCES =  \
    qoflog.c \
