--- qof-0.8.1/qof/qofdate.c.wiget	2008-06-09 13:34:52.000000000 +0200
+++ qof-0.8.1/qof/qofdate.c	2011-02-07 20:05:26.363251218 +0100
@@ -494,8 +494,11 @@ date_normalise (QofDate * date)
 		{
 			/* -1 == Dec, -4 == Sep */
 			date->qd_mon += 12 + 1;
-			date->qd_year = (date->qd_year < 0) ? 
-				date->qd_year++ : date->qd_year--;
+			if (date->qd_year < 0) {
+				date->qd_year += 1;
+			} else {
+				date->qd_year -= 1;
+			}
 		}
 	}
 	days = days_in_months[qof_date_isleap(date->qd_year)][date->qd_mon];
